mesonbuild/dependencies/base.py

```diff
@@ -693,7 +693,12 @@ def _call_pkgbin(self, args, env=None):
         sysroot = self.env.properties[self.for_machine].get_sys_root()
         if sysroot:
             env['PKG_CONFIG_SYSROOT_DIR'] = sysroot
-        new_pkg_config_path = ':'.join([p for p in extra_paths])
+        pkg_config_path_prop =  self.env.properties[self.for_machine].get_pkg_config_path()
+
+        if pkg_config_path_prop:
+          new_pkg_config_path = ':'.join([p for p in pkg_config_path_prop])
```

Comment:
__[Flake8]__ indentation is not a multiple of four ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6461#issue-5059312))

<details>

| Rule |
| - |
| E111 |

</details>
<sub>posted by Sider</sub>